app/views/shared/tags/_tag-md-color.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="tag-md">
+  <% case @beer.color.name %> 
```

Comment:
Est-ce que ça marche de tranformer `<div class="tag-icon-md blond">` en `<div class="tag-icon-md <%= @beer.color.name%>">` pour enlever tout le case ?

Reply:
J'aime cet esprit

Reply:
IT WORKS